near-contract-standards/src/fungible_token/metadata.rs

```diff
@@ -1,3 +1,4 @@
+use near_sdk::__private::schemars;
```

Comment:
1. Instead of importing it for implicit use by the derived macro, use `#[schemars(crate=…]` https://github.com/GREsau/schemars/blob/e04e3a3a8191bac1f218539733110fcc26bdbf7c/CHANGELOG.md?plain=1#L121
2. Remove the re-export in __private since we promoted it to be part of the public re-export